WordPress/Classes/Utility/ContextManager.swift

```diff
@@ -135,6 +135,41 @@ public class ContextManager: NSObject, CoreDataStack, CoreDataStackSwift {
             save(context, .asynchronously)
         }
     }
+
+    static func migrateDataModelsIfNecessary(storeURL: URL, objectModel: NSManagedObjectModel) throws {
+        guard FileManager.default.fileExists(atPath: storeURL.path) else {
+            DDLogInfo("No store exists at \(storeURL).  Skipping migration.")
+            return
+        }
+
+        guard let metadata = try? NSPersistentStoreCoordinator.metadataForPersistentStore(ofType: NSSQLiteStoreType, at: storeURL),
+            !objectModel.isConfiguration(withName: nil, compatibleWithStoreMetadata: metadata)
```

Comment:
🤦‍♂️ I mis-translated this line in #19770. Thanks for fixing this up. But I'm surprised the issue isn't caught by the migration unit tests 🤔

Reply:
Oh, thank you for the quick response! I was just about to ping you to confirm if this change was intended.